files/default/event_handler/write_chef_error_handler.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
Did you try to put this file under `libraries` folder?

In my POC I defined the custom handler under `libraries` and it was sufficient to enable it without specifying the source file.

Reply:
I didn't try the libraries folder. Will probably give it try next week.

Reply:
Sorry I paused this activity as I went back to work on the design doc last month. Now I've resumed the Chef handler activity and tried move the chef handler script to the `libraries` folder as you suggested. I confirmed that it worked and I didn't need to specifying the source file.

The code change can be found within a new PR: https://github.com/aws/aws-parallelcluster-cookbook/pull/1641/files